### DISCUSSION TITLE
image for Libre Computer Board AML-S905X-CC (Le Potato)

### DISCUSSION THREAD

**Discussion by jtmoon79** [MichaIng/DietPi#6932] [Image Request]
Can DietPi provide an image for the _Libre Computer Board AML-S905X-CC (Le Potato)_ ?

From the marketing
> Le Potato is our flagship hardware platform with support for the latest Android 9/TV, upstream Linux, u-boot, Kodi, and more.

It's [currently $30 on Amazon](https://www.amazon.com/gp/product/B074N5B8KZ/?th=1). A great low-cost high-power SBC. I'd be happy to donate costs + tip if this image is created 😁 

#### Formal device information
- Device name | Libre Computer Board AML-S905X-CC (Le Potato)
- Official product URL | https://libre.computer/products/aml-s905x-cc/
- GitHub resource URL (if available) | NA
- Image download URL | See section _Downloads_ at the Product URL link

#### Is the SBC officially supported by the Debian installer?

- yes. there's a Debian image available for it. See section _Downloads_ at the Product URL link

**Comment by Joulinar** [comment]
It's nearly impossible for us to support each and every device. However you could try to convert the existing Debian image using our installer https://dietpi.com/docs/hardware/#make-your-own-distribution

**Reply by Joulinar** [reply]
ahh probably same request as this one https://github.com/MichaIng/DietPi/discussions/5711

**Reply by isarrider** [reply]
and same as https://github.com/MichaIng/DietPi/discussions/6707

**Reply by RobertC85** [reply]
i'm willing to try to make one, if anybody has interest please contact me

**Comment by MichaIng** [comment]
Closing this as duplicate of #5711.
